src/main/User.java

```diff
@@ -0,0 +1,71 @@
+package main;
+
+import java.util.ArrayList;
+
+public class User extends Login{
+	public double average;
+	public int tuition;
+	public int preferredClassSize;
+	public double latitude;
+	public double longitude;
+		
+	public ArrayList<String> bookmarked= new ArrayList<>();
```

Comment:
This will do for now, but we'll have to add something to store the survey results down the line.